homework8/task_1.py

```diff
@@ -0,0 +1,35 @@
+s = input("enter a word: ") 
+alphabet = "abcdefghijklmnopqrstuvwxyz"
+
+new_s = s.strip()
+new_s_1 = new_s.replace(" ", "")
+new_s_2 = new_s_1.lower()
```

Comment:
აი ეს strip, lower შეგიძლია 1 ხაზში და ერთ ცვლადში მოათავსო. გთავაზობ ალტერნატივას, თუმცა შენიც სწორად მუშაობს:
text = input('Please enter some text - ').lower().strip()
new_text = ''
palindrome = True

for i in text:
    if 96 < ord(i) < 123:
        new_text += i

for i in range(int(len(new_text)/2)):
    if new_text[i] !=  new_text[-i - 1]:
        print('Is not palindrome')
        palindrome = False
        break

if palindrome:
    print('Is palindrome')